### PR TITLE
Adding OpenMined & NeurIPS Blog Posts

### PR DESCRIPTION
**Pull request by brucejlin1**

### CODE REVIEW

File: _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Additionally, to bolster this ecosystem and take the field of privacy preserving ML forward, we are also calling for contributions and supporting research efforts on this combined platform by providing funding to support the OpenMined community and the researchers that contribute, build proofs of concepts and desire to be on the cutting edge of how privacy-preserving technology is applied. We will provide funding through the [RAAIS Foundation](https://www.raais.org/), a non-profit organization with a mission to advance education and research in artificial intelligence for the common good. We encourage interested parties to apply to one or more of the fellowships listed below.
+
+# Tools Powering the Future of Privacy-Preserving ML
```
Comment (jspisak) [inline _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md]: the titles are really big. should we consider ## instead?

File: _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Development Challenges
+
+Over the coming months, we will issue regular open competitions for increasing the performance and security of the PySyft and PyGrid codebases. For performance-related challenges, contestants will compete (for a cash prize) to make a specific PySyft demo (such as federated learning) as fast as possible. For security-related challenges, contestants will compete to hack into a PyGrid server. The first to demonstrate their ability will win the cash bounty! For more information on the challenges and to sign up to receive emails when each challenge is opened, [sign up here](http://blog.openmined.org/announcing-the-openmined-pytorch-development-challenges).
+
+To apply, select one of the above projects and identify a role that matches your strengths!
+
+Cheers,
+Andrew, Laurens, Joe, and Shubho
```
Comment (jspisak) [inline _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md]: the rendered .md has the signature on the same line. maybe a double carriage return?

**Comment by jspisak** [comment]
Added some comments

File: _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md (hunk 2 above)
Comment (brucejlin1) [reply _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md]: Fixed, thanks!

File: _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md (hunk 1 above)
Comment (brucejlin1) [reply _posts/2019-12-06-openmined-and-pytorch-launch-fellowship-funding-for-privacy-preserving-ml.md]: Updated to H2 and H3

**Pull request merged**